src/vivarium_gates_nutrition_optimization/components/disease.py

```diff
@@ -105,6 +105,7 @@ def compute_disability_weight(self, index: pd.Index) -> pd.Series:
         disability_weight = pd.Series(0, index=index)
         raw_disability_weight = pd.Series(0, index=index)
         with_condition = self.with_condition(index)
+        # FIXME: this is broken (self.base_disability_weight is undefined)
```

Comment:
How has this model been working up to now then?

Reply:
@rmudambi  It appears that it's only used by the BackgroundMorbidity component which is commented out in the model spec